prost-build/src/config.rs

```diff
@@ -608,6 +609,41 @@ impl Config {
         self
     }
 
+    /// Generate the [builder pattern] API for matched message types.
+    ///
+    /// [builder pattern]: https://rust-lang.github.io/api-guidelines/type-safety.html#c-builder
+    ///
+    /// # Arguments
+    ///
+    /// **`path`** - a path matching any number of types. It works the same way as in
+    /// [`btree_map`](#method.btree_map), just with the field name omitted.
+    ///
+    /// **`builder_name`** - A name for the builder type. The struct with this name
```

Comment:
The path can match multiple messages. I think the `builder_name` should be a suffix so that it doesn't generate multiple builders with the same name.

Reply:
The builder is generated inside the submodule named after each message, so a collision could only occur with the message's nested types and enums. This should be rare enough that it can be solved by a specific matcher for messages where it occurs.

Stylistically, I prefer `my_message::Builder` to `my_message::MyMessageBuilder` or `self::MyMessageBuilder`.